x-pack/test/functional_with_es_ssl/apps/cases/group2/list_view.ts

```diff
@@ -753,8 +752,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         expect(await testSubjects.exists('all-cases-clear-filters-link-icon')).to.be(false);
       });
 
-      // FLAKY: https://github.com/elastic/kibana/issues/176716
-      describe.skip('assignees filtering', () => {
+      describe('assignees filtering', () => {
```

Comment:
What about this one? None of the changes in the `CasesTableServiceProvider` seem to be used here.

Reply:
Yes, I am not sue why they are flaky. I ran the flaky test runner a lot and I did not saw any flakyness. I suspect that it was a one time thing issue with the CI.